### PR TITLE
Better host:port splitting

### LINKED ISSUE #6: Too many colons
First, this is awesome, thanks for making a Go version of this!

But it doesn't like when I need to specify a port:

```
httpstat "http://45.56.15.67:8680/"
2016/09/23 09:03:11 unable to resolve host: too many colons in address 45.56.15.67:8680:80
```

(Believe me, I wish this was just on port 80 like every other website. But I'm having connectivity issues to it which is why I thought I'd try httpstat, ha.)

### PR DESCRIPTION
**Pull request by mholt**
Fixes #6

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -85,20 +85,28 @@ func main() {
...
 		default:
 			log.Fatalf("unsupported url scheme %q", scheme)
-			return "", 0 // not reached
 		}
+		return host, port
 	}()
 
 	t0 := time.Now() // before dns resolution
-	raddr, err := net.ResolveTCPAddr("tcp", fmt.Sprintf("%s:%d", host, port))
+	raddr, err := net.ResolveTCPAddr("tcp", fmt.Sprintf("%s:%s", host, port))
```
Comment (mholt) [inline main.go]: Actually I should probably use `net.JoinHostPort()` here.

**Comment by davecheney** [comment]
Nice, do you want to add some tests to the .travis.yml file, something like
```bash
httpstat http://dave.cheney.net:80/
```

**Comment by mholt** [comment]
Sure. As soon as I get back from the store. :car:

**Comment by davecheney** [comment]
I'll do the test

**Pull request merged**